client/deposits/details/index.tsx

```diff
@@ -166,7 +171,7 @@ export const DepositOverview: React.FC< DepositOverviewProps > = ( {
 					}
 				>
 					{ () => [
-						depositDateItem,
+						<DepositDateItem key="dateItem" deposit={ deposit } />,
```

Comment:
I wonder what's the `key="dateItem"` prop is for because I see `DepositDateItem` only reads the `deposit` prop.

Reply:
I learned about this during a pair-coding session with @haszari . React requires you to add a unique key to any component within a list to make sure it re-renders correctly. [Ref. article.](https://kentcdodds.com/blog/understanding-reacts-key-prop)